pkg/core/storage/boltdb_store.go

```diff
@@ -0,0 +1,129 @@
+package storage
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/CityOfZion/neo-go/config"
+	"github.com/boltdb/bolt"
+	"github.com/syndtr/goleveldb/leveldb/util"
+)
+
+var (
+	Bucket = []byte("DB") // Bucket name
+)
+
+// BoltDB is the storage implementation for storing and retrieving
```

Comment:
`golint` won't be happy with this.

And it's not:
https://circleci.com/gh/nspcc-dev/neo-go/3393?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link